app/assets/javascripts/models/tour.js

```diff
@@ -0,0 +1,51 @@
+import Tether from 'tether';
+import Shepherd from 'tether-shepherd';
+
+module.exports = Backbone.Model.extend({
```

Comment:
Does it _has_ to be a Backbone model? Can't it be a simple class? I don't see where it's using Backbone features.